plugins/modules/s3_bucket.py

```diff
@@ -286,6 +307,14 @@
     name: mys3bucket
     state: present
     acl: public-read
+
+# Default Object Lock retention
+- amazon.aws.s3_bucket:
+    name: mys3bucket
+    state: present
+    object_lock_default_retention:
+      mode: governance
+      days: 1
 """
 
 RETURN = r"""
```

Comment:
We should update the RETURN block documentation to include the new `"object_lock_default_retention"` value.